lib/stat_tracker.rb

```diff
@@ -37,43 +37,153 @@ def lowest_total_score
 
   def percentage_home_wins
     home_wins = 0
-    away_wins = 0
+    visitor_wins = 0
+    ties = 0
     CSV.foreach(game_teams, headers: true, header_converters: :symbol) do |row|
```

Comment:
Good use of converters into symbols! Watch out for spec harness on their format whenever you run spec harness.